acceptance/tests/resource/service/launchd_provider.rb

```diff
@@ -0,0 +1,82 @@
+test_name 'Mac OS X launchd Provider Testing'
+ 
+confine :to, {:platform => /osx/}, agents
```

Comment:
combining platform and agents here might not do what we think.  i haven't tried it yet...